bin/wasm-node/javascript/demo/demo.mjs

```diff
@@ -17,7 +17,7 @@
 
 // This file launches a WebSocket server that exposes JSON-RPC functions.
 
-import * as smoldot from '../dist/index.js';
+import { start } from '../dist/mjs/index.js';
```

Comment:
Does this PR break `import * as smoldot from '....'`?

Reply:
No it does not.. Will revert